src/django/pfb_analysis/serializers.py

```diff
@@ -98,6 +98,37 @@ class Meta:
         read_only_fields = fields
 
 
+class AnalysisLocalUploadTaskSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = AnalysisLocalUploadTask
+        fields = ('uuid', 'error', 'job', 'status', 'upload_results_url',)
```

Comment:
Since these show up in this order on the detail page, putting 'status' before 'error' makes it a bit more readable--it's natural to look below the ERROR status for the message, and when the status isn't ERROR having it first avoids having to look below to figure out why the "local upload task error" field is blank.

Also, having 'job' included in the embedded serialization and putting it on the detail page as "local upload task job" is a bit odd.  It took me a moment to realize that's not some other related entity, just a repetition of the ID of the job whose detail page I'm looking at.  I guess that's related to
> Also adds the ability to filter the import task endpoint directly by analysis job UUID, which is currently unused by the front-end site.

How hard would it be to suppress that field in the embedded serialization?

Reply:
I missed the `job` was coming through nested, good catch. I'll split out the serializer into two versions.